_nodes/0011.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Discrete right fibrations
+macrolib: topos
+---
+
+A displayed category $E$ over $B$ is a *discrete right fibration* when for each
+morphism $f:x\to y$ and displayed object $\bar{y}\in E_y$, there exists a unique pair
+of $\bar{x}\in E_x$ and $\bar{f}:\bar{x}\to\Sub{f}\bar{y}$.
+«
+\DiagramSquare{
+  west/style = lies over,
+  east/style = lies over,
+  height = 1.5cm,
+  nw = \bar{x},
+  ne = \bar{y},
+  north = \bar{f},
+  sw = x,
+  se = y,
+  south = f,
+}
+»
+
+TODO: need to require that each $E_x$ is small?
+
+Whereas cartesian fibrations correspond to categories indexed pseudofunctorially by a
```

Comment:
It might be good to actually state this elsewhere in the discussion of cartesian fibrations and then refer back to it here, as it is an important intuition that we haven't really brought to the surface. We will have to bring up the meta-bicategory of all categories, and we haven't explained what a (meta-)bicategory is, but we can just link to the nlab or something.